languages/Spanish

```diff
@@ -0,0 +1,134 @@
+38=No se pudo encontrar el archivo de opciones. Se usará la configuración predeterminada. Guarde sus opciones desde el menú de opciones para desactivar este mensaje.
+39=Nivel 1
+40=sí
+41=no
+42=Protectores de cuchillas
+43=trampas explosivas
+44=vidas
+45=Puntos
+46=Pociones
+47=corriendo
+48=caminar
+49=Salud:
+50=. Armadura:
+51=munición
+52=¿Quieres salir? Presiona Intro para salir, o espacio para volver al juego.
+53=Perdiste una vida!
+54=Nivel
+55=le ganaste al jefe!
+56=Gastaste
+57= segundos en este nivel, lo que resulta en una bonificación de tiempo de
+58=.
+59=La puntuación para este nivel fue
+60=. Ahora tienes
+61= puntos totales.
+62=nivel 6
+63=espada!
+64=arma!
+65=trampas explosivas!
+66=ametralladora!
+67=puntos extra!
+68=vida extra!
+69=comprar vidas extra, 4000 puntos por vida
+70=Comprar pociones, 1000 puntos por poción
+71=comprar munición, 900 puntos por cada paquete de 30 municiones
+72=comprar trampas explosivas, 1200 puntos por cada conjunto de 5 trampas explosivas
+73=comprar armadura, 800 puntos por cada armadura
+74=comprar artículos de hojas de evasión, 1500 puntos por artículo
+75=¡Bienvenido a la tienda de la zona de batalla! Tienes 
+76= puntos que me puedes dar! ¡Seleccione un artículo que desea comprar!
+77=paquetes de munición
+78=conjuntos de trampas explosivas
+79=clasificaciones de armadura
+80=¿Cuánto del artículo quieres comprar?
+81=así, Tú quieres comprar 
+82=. espero que tengas 
+83= puntos para darme, porque eso es lo que costará. ¿Aún quieres comprarlo?
+84=¿Qué crees que estás haciendo? ¿Estás tratando de engañar al tendero? Bueno, no tienes suficientes puntos para comprar cosas. Lo siento, vuelve más tarde, ¡y no me vuelvas barato la próxima vez!
+85=oh, espera, estás jugando en el modo de realidad. Lo siento, pero estamos agotados en vidas extra. Aquí está tu dinero bac.
+86=¡ya no tienes más objetos para evadir cuchillas!
+87=explosión aburrida
+88=asesinato en masa medio
+89=incendio premeditado avanzado
+90=¡Solo quiero morir ahora!
+91=¡Elige tu nivel de dificultad!
+92=trucos para la zona de batalla!	Para activar un truco, solo escribe el comando en el juego. Trabajan la mayoría de las veces.	Mantén presionada la tecla Eliminar y presiona z: da 250 armaduras	Alt + barra invertida: evade las cuchillas para siempre	Control + f9: salta este nivel y pasa al siguiente	espacio de espera y presiona 9: ¡te da una gran cantidad de munición!	Mantenga presionada la tecla f4 y presione enter: ¡le da 250 vidas extra!	Mantenga presionada la tecla enter y presione x: ¡le da 500 pociones más!	Mantén presionada u, luego mantén presionado el espacio y luego presiona el desplazamiento a la izquierda: ¡te dan 500 trampas explosivas adicionales!
+93=Por ganar más de 20000 puntos en el juego, ¡acabas de recibir un archivo de trucos! Se puede encontrar en el directorio de su zona de batalla.
+94=salir del laboratorio
+95=escenas del país, quiero decir ciudad, vida
```

Comment:
Country is not like a state here, it's like village, rural life. So I would play with aldeano y urbano or something like this :)